src/components/ManhattanWithTooltip.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { ListTooltip } from 'ot-ui';
+import { Manhattan } from 'ot-charts';
+
+import { tableColumns } from './ManhattanTable';
+
+class ManhattanWithTooltip extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      anchorEl: null,
+      anchorData: null,
+      open: false,
+    };
+    this.handleMouseover = this.handleMouseover.bind(this);
+    this.handleMouseout = this.handleMouseout.bind(this);
```

Comment:
We can remove these bindings and use class property notation which is supported by create-react-app.